setup.cfg

```diff
@@ -1,3 +1,49 @@
+[metadata]
+name = dkist
+description = DKIST User Tools
+long_description = file: README.rst
+author = "NSO / AURA"
+author_email = "stuart@cadair.com"
+license = BSD 3-Clause
+url = http://dkist.nso.edu
+edit_on_github = True
+github_project = DKISTDC/dkist
+classifiers =
+  'Programming Language :: Python'
+  'Programming Language :: Python :: 3'
+  'Programming Language :: Python :: 3.6'
+  'Programming Language :: Python :: 3.7'
+
+[options]
+python_requires = >=3.6
+setup_requires = setuptools_scm
```

Comment:
I'd just get rid of this and include it in the toml file

Reply:
That would enforce pip 19? or only 18?

Reply:
Oh also, this would break `sdist` so that would be no good.